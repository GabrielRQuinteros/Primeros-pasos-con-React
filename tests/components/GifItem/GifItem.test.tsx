import { render } from '@testing-library/react';
import GifItem from '../../../src/components/GifItem/GifItem';

describe('Pruebas del componente: <GifItem/>', () => {
    
    const title: string = 'One punch Man Test';
    const url: string = 'https://alfabetajuega.com/hero/2025/05/saitama.1746780301.4579.jpg?width=1200&aspect_ratio=16:9';

    test('1- Renderizacion del Componente con parametros correctos', () => {
        const { container } = render( <GifItem title={title} url={url}/> );
        expect( container ).toMatchSnapshot();
    });

    
  
})
