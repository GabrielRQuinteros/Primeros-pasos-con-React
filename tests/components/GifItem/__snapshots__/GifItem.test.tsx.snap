// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente: <GifItem/> 1- Renderizacion del Componente con parametros correctos 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="One punch Man Test"
      src="https://alfabetajuega.com/hero/2025/05/saitama.1746780301.4579.jpg?width=1200&aspect_ratio=16:9"
    />
    <p>
      One punch Man Test
    </p>
  </div>
</div>
`;
